src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2

```diff
@@ -101,7 +102,12 @@ module "server__{{ server_name }}" {
   secondary_volume_size = {{ server_spec.block_device.volume_size|default(0)|tojson }}
   secondary_volume_type = {{ server_spec.block_device.volume_type|default("")|tojson }}
   secondary_volume_encrypted = {{ server_spec.block_device.encrypted|default(False)|tojson }}
+
+{% if server_name == "control1-production" %}
```

Comment:
ha! Oddly specific, but since it's a temporary thing we don't think we'll need again, and which we'll remove after, this does seem like a fine way to do it